### DISCUSSION TITLE
Hot reload /  View Preview?

### DISCUSSION THREAD

**Discussion by sebastianvirlan-plt** [sweetpad-dev/sweetpad#13] [Any discussions (ideas, questions, ...)]
Hi, are there any plans to implement a Preview mode for View files similar to Xcode offer? Amazing project btw.

**Comment by hyzyla** [comment]
Copied answer from Reddit:

> Swift preview doesn’t work because it is hard to reverse engineer, and I haven't found an easy way to render or execute them. I think you can configure hot reload by using a third-party tool like Inject [1], but the extension currently does not have any functionality for hot reloading.

> In the future, I think I can build a tool that gathers all tools into one CLI that seamlessly integrates with SweetPad, including some third-party tools for hot reloading

1.	⁠https://github.com/krzysztofzablocki/Inject
